arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+
+To read the child chain parameters, you can use these getter functions:
+
+- `getAllChainOwners`: Provides the list of all current chain owners.
+
+- `getNetworkFeeAccount`:  Returns the network fee account address.
+
+- `getInfraFeeAccount`:  Returns the infrastructure fee account address.
+
+- `isChainOwner`:  Allows you to check whether an address is on the list of chain owners.
+
+### Configuring the Child Chain Using the Orbit SDK
+
+In the Orbit SDK, we use the [Client Extension](https://viem.sh/docs/clients/custom#extending-with-actions-or-configuration) feature of Viem to extend the public client. In the Orbit SDK, we defined `arbOwnerPublicActions` to use it and extend the client on Viem. An example of creating a public client extended with arbOwner public actions is:
+
+
+```js
+import { createPublicClient, http } from 'viem';
+import { arbOwnerPublicActions } from '@arbitrum/orbit-sdk';
+
+const client = createPublicClient({
+  chain: arbitrumLocal,
+  transport: http(),
+}).extend(arbOwnerPublicActions);
+```
+
+With `arbOwnerPublicActions` and the public client in the Orbit SDK, we've added two new methods to the public clients:
+
+
+#### 1. arbOwnerReadContract 
+
+This method can be used to read the parameters of the child chain discussed in the [previous section](#getter-functions). An example of using this method with the `client` defined in the previous section is:
+
+```js
+  const result = await client.arbOwnerReadContract({
+    functionName: 'getAllChainOwners',
+  });
+```
+
+Changing the function names in the list in this [section](#getter-functions) will give you the other parameters.
+
+#### 2. arbOwnerPrepareTransactionRequest
+
+This method can be used to configure the parameters on the ArbOwner precompile, which are listed in this [section](#setter-functions). An example of utilizing this method to configure parameters using the `client` defined in the previous section is:
+
+```js
+  // Adding a random address as chain owner using the upgrade executor
+  const transactionRequest = await client.arbOwnerPrepareTransactionRequest({
+    functionName: 'addChainOwner',
+    args: [randomAccountAddress],
+    upgradeExecutor: false,
+    account: owner.address,
+  });
+
+  // Submitting the transaction to add a chain owner
+  await client.sendRawTransaction({
+    serializedTransaction: await owner.signTransaction(transactionRequest),
+  });
+```
+
+To use this method, as shown in the example above, some inputs need to be defined. `functionName` is the name of the method you want to use to set the parameter, which can be found in [this section](#setter-functions). `args` are the arguments for the defined function. The `upgradeExecutor` field specifies whether an `upgradeExecutor` contract governs your chain. If it is not using an `upgradeExecutor`, you can set it to `false`, similar to the example above. Also, the `account` field defines the chain owner if an `upgradeExecutor` does not govern the chain.
```

Comment:
```suggestion
To use this method as shown in the example above, some inputs need to be defined:

| Parameter        | Description                                                                                                                                                                 |
|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `functionName`   | The name of the method you want to use to set the parameter, which can be found in [the Setter functions section](#setter-functions).                                        |
| `args`           | The arguments for the defined function.                                                                                                                                     |
| `upgradeExecutor`| Specifies whether an `upgradeExecutor` contract governs your chain. If it is not using an `upgradeExecutor`, you can set it to `false`, similar to the example above.       |
| `account`        | Defines the chain owner if an `upgradeExecutor` does not govern the chain.                                                                                                  |
```

- nit - specific link labels ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))
- tables == happiness